src/lib/components/InfoTooltip.svelte

```diff
@@ -38,7 +38,7 @@
 				<strong class="block text-balance text-base font-semibold leading-5">{title}</strong>
 			{/if}
 			{#if description}
-				<p class="text-pretty text-sm leading-4 text-muted-foreground">{description}</p>
+				<p class="text-pretty text-sm leading-4 text-muted-foreground">{@html description}</p>
```

Comment:
This was needed to make this happening in a nicely formatted way:

![image](https://github.com/user-attachments/assets/39c87d46-77a6-4ced-a6ae-d878f160145d)